src/index.ts

```diff
@@ -244,3 +244,6 @@ function merge<T>(target: T, ...sources: unknown[]): T {
     }
     return output as T;
 }
+
+// Export Zod factory functionality
+export { createFactoryFromZod, type ZodFactoryConfig } from './zod-factory.js';
```

Comment:
this is problematic - we want to have zod as an optional dependency. This means for those who dont have it, this will still be imported, which means any zod imports will throw. To avoid this, remove this re-export from here. 

Users will have to import directly from zod-factory.

I would also rename "zod-factory" to "zod" so users can import from "interface-forge/zod"